src/analysis/calculus/fderiv_measurable.lean

```diff
@@ -13,7 +13,7 @@ import measure_theory.borel_space
 In this file we prove that the derivative of any function with complete codomain is a measurable
 function. Namely, we prove:
 
-* `is_measurable_set_of_differentiable_at`: the set `{x | differentiable_at 𝕜 f x}` is measurable;
+* `measurable_set_set_of_differentiable_at`: the set `{x | differentiable_at 𝕜 f x}` is measurable;
```

Comment:
Can you search and replace `measurable_set_set` with `measurable_set`?

Reply:
Isn't this naming deliberate, as this is about `measurable_set (set_of _)`?